src/assets/css/_fonts.scss

```diff
@@ -0,0 +1,16 @@
+@import url(https://fonts.googleapis.com/css?family=Roboto:100,100italic,300,300italic,400,400italic,500,500italic,700,700italic,900,900italic);
+@import url(https://fonts.googleapis.com/css?family=Roboto+Mono:100,100italic,300,300italic,400,400italic,500,500italic,700,700italic);
+
+// Add new font properties here.
+// The font property variable should be in the form $font-<type|size|...>-<domain>[-subdomain]
+
+$font-type-default: 'Roboto', 'Verdana', sans-serif;
+$font-type-code: 'Roboto Mono', 'Curier New', monospace;
+
+$font-size-default: 15pt;
+$font-size-code: 13pt;
+$font-size-header: 15pt;
+$font-size-footer: 15pt;
+$font-size-title: 15pt;
```

Comment:
Hva er grunnen til at du lager variabler for header, footer og title, som alle er like som `$font-size-default`?
Regner du med at disse blir endret individuelt senere?